lms/resources/_js_config.py

```diff
@@ -144,6 +144,8 @@ def enable_lti_launch_mode(self):
             "allowedOrigins": self._request.registry.settings["rpc_allowed_origins"]
         }
 
+        self._canvas_settings()
```

Comment:
This section of the config is used when launching an assignment, whereas the information about whether groups are enabled and the metadata to make an API call to list group categories are needed in assignment selection mode, known as "content item selection mode" in the context of this module. I believe that terminology comes from the LTI spec. See the `enable_content_item_selection_mode` method.